config/settings.py

```diff
@@ -148,3 +148,16 @@
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'frontend', 'src', 'static'),
 )
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'assets')
```

Comment:
this is  not where static root should be. All assets need to be inside of frontend/src/static/assets